onnxruntime/contrib_ops/cpu/layer_norm.cc

```diff
@@ -15,7 +15,7 @@ namespace contrib {
   ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
       LayerNormalization,                                         \
       kOnnxDomain,                                                \
-      1,                                                          \
+      9,                                                          \
```

Comment:
Why it is 9?

Reply:
Here is previous conversation:

Scott McKay
Are there any models with opset < 9 that potentially use this op, or is this change just making the SinceVersion correct?

Sherlock Huang
LayerNorm should actually by in kMsDomain, since it's not part of ONNX spec yet.
Currently, LayerNorm op is only introduced through layer_norm_fusion.
So, in theory, not model should be affected

Sherlock Huang
We are updating the opset to 9, coz training only support onnx opset 9+.